docs/common/help.go

```diff
@@ -11,6 +11,11 @@ func GetGlobalEnvVars() string {
 		If set to ERROR, JFrog CLI logs error messages only.
 		It is useful when you wish to read or parse the JFrog CLI output and do not want any other information logged.
 
+	JFROG_CLI_TIMESTAMP
+		[Default: TIME]
+		This variable determines the timestamp prefix appearance of JFrog CLI's logs.
```

Comment:
This variable determines the timestamp prefix appearance of JFrog CLI's logs.
-->
Controls the log messages timestamp format.